docs/messaging/messaging-spans.md

```diff
@@ -216,20 +216,14 @@ Span kind SHOULD be set according to the following table, based on the operation
 | Operation type | Span kind|
 |----------------|-------------|
 | `create`       | `PRODUCER` |
-| `publish`      | `PRODUCER` if the context of the "Publish" span is used as creation context. |
+| `publish`      | `PRODUCER` if the context of the "Publish" span is used as creation context, otherwise `CLIENT`. |
 | `receive`      | `CONSUMER` |
```

Comment:
I think it should be `CLIENT`

https://github.com/open-telemetry/semantic-conventions/issues/1366

Reply:
updated